components/vf-global-header/vf-global-header.scss

```diff
@@ -44,6 +44,14 @@ $vf-global-header__link-text--color: ui-color(black);
 // This fixes that:
 // Note: we duplicate the top margin that's already on line 22.
 
+// These are for static html pages
+link {
```

Comment:
This would affect all contentHub imports. 

Perhaps it's better if this css lives within `vf-stack.scss`? or within `vf-content-hub-import` and scope it as: `.vf-stack link[data-embl-js-content-hub-loader]` 

_Note: that the `link > div` relationship on contentHub imports isn't always assured._

Aside: I don't think `link > div` ever happens as the HTML gets pulled out of the import.

Reply:
I think keeping it closer to where 'the problem is' is probably better

so not `vf-stack` but maybe the `vf-content-hub-import` would be better placed.

Reply:
@khawkins98 leaving this one for you to resolve conversation and I'll merge when approved